packages/core/src/__tests__/overall.test.ts

```diff
@@ -161,9 +161,13 @@ describe("Energy API", () => {
     if (res.isOk()) {
       // NOTE: delibrately not cache the overall objective and re-generate for original and filtered states
       const state = res.value;
-      const smallerThanFns = state.constrFns.filter(
-        (c) => c.ast.expr.name.value === "smallerThan"
-      );
+      const smallerThanFns = state.constrFns.filter((c) => {
+        if (c.ast.expr.body.tag === "FunctionCall") {
+          return c.ast.expr.body.name.value === "smallerThan";
```

Comment:
The PR description says `lessThan`; is there a difference between that and `smallerThan`?

Reply:
Hmmm oops I need to look more into this ...

I think this test case is specifically looking for `smallerThan` not `lessThan`. `smallerThan` takes two shapes, whereas `lessThan` takes two numbers.

Reply:
Fixed - now it is only looking for `smallerThan`.